.github/actions/pull_images/action.yml

```diff
@@ -35,4 +19,4 @@ runs:
 
     - name: Pull
       shell: bash
-      run: docker compose pull
+      run: docker compose pull -q
```

Comment:
why the `-q` ?

Reply:
The output of the command unfortunately prints progress very quickly instead of in stages, which ends up filling up the log with tons and tons of lines.